frontend/src/utils.js

```diff
@@ -0,0 +1,51 @@
+// import { refreshToken } from './auth';
```

Comment:
Just double-checking. Commented code in `utils.js` is intended to be used in the near future, maybe while implementing sign-in, isn't it?

Reply:
Yep, I'm praying I can use it after that heh